src/main/common/interest.ts

```diff
@@ -15,17 +16,13 @@ export async function getInterestDetails (claim: Claim): Promise<object> {
   if (claim.claimData.interestDate.type === InterestDateType.CUSTOM) {
     interestDate = claim.claimData.interestDate.date
   } else {
-    interestDate = claim.createdAt
+    interestDate = claim.issuedOn.startOf('day')
```

Comment:
Is that needed? I think `issuedOn` should only have date information, no time there.